.mailmap

```diff
@@ -0,0 +1 @@
+Amethyst Reese <amethyst@n7.gg> <john@noswap.com>
```

Comment:
Does this format support comments? It would be good to add a comment briefly explaining what the file does, maybe pointing to the link you provided in the PR description. For future readers of the repo, it may not be obvious what this file does, as unlike other git-related files it doesn't have ".git" in the name.

Reply:
Just confirmed locally that it does work.
```suggestion
# This file sets the canonical name for contributors to the repository.
# Documentation: https://git-scm.com/docs/gitmailmap
Amethyst Reese <amethyst@n7.gg> <john@noswap.com>
```